### ISSUE TITLE
Can not add peers after installed

### ISSUE THREAD

**Issue by netcookies** [wg-dashboard/wg-dashboard#23]
There is not table row to input peers args, but an Add Peers button.
It just say "could not save user;" when I click Add Peers button. Then I was force to re-signin.
![image](https://user-images.githubusercontent.com/224964/65816856-8c813c00-e233-11e9-9be8-5990f9d55e62.png)

**Comment by daluf** [comment]
Hi, is this still an problem occuring to you? Did you (re-)start wireguard through the dashboard after installing?

**Comment by LaurenceMillward** [comment]
@daluf I'm also having this issue, kinda. Fresh install of Ubuntu 18, ran the install script, everything appears to be in order but adding a peer throws `could not save user: COULD_NOT_ADD_PEER_TO_wg0`

Edit: Yeah, after the install script ran the first thing I did was reboot the server. I've also added the external IP to the dashboard  & saved the config / restarted via that. 

However if I check the contents of sudo nano /etc/wireguard/wg0.conf the peer does actually exist.. So it is saving it but the web dashboard is saying otherwise. 

Edit2: I can also connect fine so the dashboard is doing what it should be doing, everything works as expected.. Other than the error which isn't an error.
